### DISCUSSION TITLE
JWS into a numeric QR

### DISCUSSION THREAD

**Discussion by Fergusgus** [smart-on-fhir/health-cards#202] [Q&A]
hello,
Need help converting JWS into a numeric QR.
I'm new with github and no knowledge of js and json

**Comment by christianpaquin** [comment]
This [wiki page](https://github.com/smart-on-fhir/health-cards/wiki/Libraries-for-SMART-Health-Cards) lists libraries to deal with QRs. What language are you targettng?

**Reply by Fergusgus** [reply]
English

**Reply by christianpaquin** [reply]
I meant programming language. I assume it won't be javascript, per your initial comment.

**Reply by Fergusgus** [reply]
Sorry about that, lol. I have installed python now and also Javascript.
I will read through the wiki page and if I'm stuck I will get back.
